frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/homepage/MyProjectsPortlet.java

```diff
@@ -230,4 +244,29 @@ private String acceptedReleases(ReleaseClearingStateSummary releaseClearingState
                 p -> p.getReleaseClearingStateSummary() != null ? p.getReleaseClearingStateSummary().approved : -1);
         return isAscending ? comparator : comparator.reversed();
     }
+    
+    private List<Project> getWithFilledClearingStatus(List<Project> projects, Map<String, Boolean> clearingState ) {
+
+        if (clearingState != null && !clearingState.isEmpty()) {
```

Comment:
`!CommonUtils.isNullOrEmptyMap(clearingState);` can be used here

Reply:
added the CommonUtils to check the condition.